3d-venues-and-indoor-routing/app/src/main/AndroidManifest.xml

```diff
@@ -18,15 +18,15 @@
 
         <!--Developers should put application credentials here.To obtain them, please register the application
          at https://developer.here.com/develop/mobile-sdks-->
-         <meta-data
-         android:name="com.here.android.maps.appid"
-         android:value="{YOUR_APP_ID}" />
-         <meta-data
-         android:name="com.here.android.maps.apptoken"
-         android:value="{YOUR_APP_CODE}" />
-         <meta-data
-         android:name="com.here.android.maps.license.key"
-         android:value="{YOUR_LICENSE_KEY}" />
+        <meta-data
+        android:name="com.here.android.maps.appid"
```

Comment:
Sorry I think I mis-represented, what I meant was we should still use the original format i.e 
    <meta-data 
         android:name=......
    <meta-data
         android:name......

But without any extra space at front.